pkg/machines/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -76,6 +76,7 @@ const StoragePoolTypeRow = ({ onValueChanged, dialogValues }) => {
         { type: 'dir', detail: _("Filesystem Directory") },
         { type: 'netfs', detail:_("Network File System") },
         { type: 'iscsi', detail: _("iSCSI Target") },
+        { type: 'iscsi-direct', detail: _("iSCSI direct Target") }
```

Comment:
Do we need to be concerned about running on OSes with an earlier libvirt that doesn't support this type yet?

Reply:
I was thinking about it as well. So this is going to be available inly for distros with libvirt-dbus.

Thus fedora > 27, debian-testing and ubuntu 18.04 and rhel8
From the above only fedora > 28 debian-testing have libvirt >= 4.7

In my opinion, I think it's not such a big deal to just have it in without any libvirt version check. In case someone has older libvirt, creation of this pool will just fail, giving some error of unsupported type to the UI or something. It's not that it will crash or anything.

And since this pool type it's quite not so common, I would expect for someone who wants to use it to know what they are doing.

Reply:
@martinpitt  wdyt?

Reply:
Thanks, seems safe enough then.